### PR TITLE
validate ENV values at run-time

### LINKED ISSUE #1156: validate ENV values at container start time
- [ ] download all external assets whose URLs are passed in ENV values and store them in `/public` folder
- [ ] validate ENV values when a container is starting

### PR DESCRIPTION
**Pull request by tom2drum**
resolves #1156

### CODE REVIEW

File: configs/envs/.env.eth [hunk 1]
```diff
@@ -36,7 +36,7 @@ NEXT_PUBLIC_GRAPHIQL_TRANSACTION=0xf7d4972356e6ae44ae948d0cf19ef2beaf0e574c18099
 NEXT_PUBLIC_API_SPEC_URL=https://raw.githubusercontent.com/blockscout/blockscout-api-v2-swagger/main/swagger.yaml
 NEXT_PUBLIC_HAS_BEACON_CHAIN=true
 NEXT_PUBLIC_IS_ACCOUNT_SUPPORTED=true
-NEXT_PUBLIC_AUTH_URL=http://localhost:3000
+# NEXT_PUBLIC_AUTH_URL=http://localhost:3000
```
Comment (isstuev) [inline configs/envs/.env.eth:39]: ?

File: deploy/tools/envs-validator/schema.ts [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+    NEXT_PUBLIC_MIXPANEL_PROJECT_TOKEN: yup.string(),
+  })
+  .concat(accountSchema)
+  .concat(adsBannerSchema)
+  .concat(marketplaceSchema)
+  .concat(rollupSchema)
+  .concat(beaconChainSchema)
+  .concat(sentrySchema);
+
+export default schema;
```
Comment (isstuev) [inline deploy/tools/envs-validator/schema.ts:339]: 😢

**Review by isstuev — APPROVED** [review]

File: deploy/tools/envs-validator/schema.ts (hunk 2 above)
Comment (tom2drum) [reply deploy/tools/envs-validator/schema.ts:339]: I can split it into separate files, if it seems more readable for you

File: configs/envs/.env.eth (hunk 1 above)
Comment (tom2drum) [reply configs/envs/.env.eth:39]: atm, yup thinks that localhost url is not valid url, so I have to comment this out.. auth doesn't work loacally anyways

**Pull request merged**